libcalico-go/lib/names/policy.go

```diff
@@ -21,12 +21,14 @@ import (
 )
 
 const (
-	DefaultTierName            = "default"
-	AdminNetworkPolicyTierName = "adminnetworkpolicy"
+	DefaultTierName                    = "default"
+	AdminNetworkPolicyTierName         = "adminnetworkpolicy"
+	BaselineAdminNetworkPolicyTierName = "baselineadminnetworkpolicy"
 
-	K8sNetworkPolicyNamePrefix      = "knp.default."
-	K8sAdminNetworkPolicyNamePrefix = "kanp.adminnetworkpolicy."
-	OssNetworkPolicyNamePrefix      = "ossg."
+	K8sNetworkPolicyNamePrefix              = "knp.default."
+	K8sAdminNetworkPolicyNamePrefix         = "kanp.adminnetworkpolicy."
+	K8sBaselineAdminNetworkPolicyNamePrefix = "kbanp.adminnetworkpolicy."
```

Comment:
This prefix should be `kbanp.baselineadminnetworkpolicy`. This is long, but basically the second part `baselineadminnetworkpolicy` is the tier name.

Reply:
If we want to make it shorter, maybe we can name the tier `baselinepolicy` and this prefix `kbanp.baselinepolicy`.

Reply:
Good spot, thanks.